### PR TITLE
Get cloudwatch widget images for instances

### PR DESCRIPTION
**Pull request by fishnix**
This returns a simple 1 day widget image per metric.  We will need to implement caching since it does have cost associated.

### CODE REVIEW

**Comment by damnski** [comment]
:+1:

File: cloudwatch/errors.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+			// Access forbidden.
+			"Forbidden",
+
+			// The AWS access key ID you provided does not exist in our records.
+			"InvalidAccessKeyId":
+
+			return apierror.New(apierror.ErrForbidden, msg, aerr)
+		case
+			// The specified bucket does not exist.
```
Comment (tenyo) [inline cloudwatch/errors.go]: 👀 `bucket`

File: cloudwatch/metrics.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+			[]string{
+				"AWS/EC2",
+				metric,
+				"InstanceId",
+				id,
+			},
+		},
+		Period: 300,
+		Start:  "-P1D",
+		End:    "PT0H",
```
Comment (tenyo) [inline cloudwatch/metrics.go]: Eventually we'd like to specify these in the input somehow

**Comment by tenyo** [comment]
Looks good 👍 I'll try it in test

**Pull request merged**